### ISSUE TITLE
Cannot access Outlook profile recipient.Name [0x80004004]

### ISSUE THREAD

**Issue by duplesy** [phw198/OutlookGoogleCalendarSync#909]
**OGCS Version**: v2.8.2.0 (Alpha)  
**Edition**: Installed
**Sync Direction**: G->O / O->G / O<->G

**Bug Description**  
OGCS crashes and the process thread closes by itself

**Steps to Reproduce the Issue**  
1. Launch OGCS 

Any additional information that may help troubleshoot the issue.
Version of outlook: 15.0.5172.1000 MSO 32-bit
Machine: Surface Pro
Windows version: Windows 10 Enterprise 1809 version

[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/3904287/OGcalsync.log)
:paperclip: Attach your `OGcalsync.log` file.

**Comment by phw198** [comment]
Please could you increase your logging level to `FINE` - you'll need to manually open the `settings.xml` file in a text editor (eg notepad) and change the line `<LoggingLevel>DEBUG</LoggingLevel>`.

Then open OGCS to reproduce the problem and upload your logfile again.

Thanks.

**Comment by duplesy** [comment]
Hi

Attached is the updated log file.

Look forward to the a quick solve :)

Many thanks

Rgds
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/3907865/OGcalsync.log)

**Comment by phw198** [comment]
Seems like either you're running OGCS in a very restricted work environment, or have very aggressive anti-virus perhaps. At any rate, it's not letting the application access the name associated with your Outlook profile (I've seen AV block accessing the email address, but never name yet).

:package: Anyway, try out [v2.8.2.16.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/3936099/v2.8.2.16.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)) and see if that gets further?

**Comment by duplesy** [comment]
Hi

After running this for the first time, the symantec antivirus and windows defender catch it. After allowing it in both, it doesnt even go as far as updating the log.

Any other way I can help debug this? The current fix didnt help.

Rgds

**Comment by phw198** [comment]
So what _does_ happen after allowing it in AV?

If it just crashes immediately, I would strongly suspect you've failed to apply the hotfix at all, and are simply trying to run it as a standalone executable...

**Comment by duplesy** [comment]
Hey there

I applied the patch correctly and the log got updated. The fix dint work but the log should be helpful to you. (Attached)

Given the log, is there something specific information that I could help you with, to help you debug?

Let me know
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/3941982/OGcalsync.log)

**Comment by phw198** [comment]
:package: OK, please try again with [v2.8.2.23.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/3942265/v2.8.2.23.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).

For my info, can you describe the environment you're running in? Looks like a corporate Exchange account - is the AV also managed by your IT? Can you temporarily disable the AV?

**Comment by duplesy** [comment]
Works well. It raised an AV exception, but once I allowed it - it went through. 

I have a corporate exchange account and AV is managed by the IT. I couldnt disable the AV but I was able to allow exceptions and exclude OGCS to .

Hope you will incorporate the fix in the main build. I have been using OGCS for a while now, just moved to a new laptop and the issues started.

Sync just finished and worked flawlessly as before. Thanks for the help
